modules/nw-multus-bridge-object.adoc

```diff
@@ -23,50 +23,78 @@ plugin:
 
 |`isGateway`
 |`boolean`
-|Set to `true` to assign an IP address to the bridge. The default value is `false`.
+|Optional: Set to `true` to assign an IP address to the bridge. The default value is `false`.
 
 |`isDefaultGateway`
 |`boolean`
-|Set to `true` to configure the bridge as the default gateway for the virtual network. The default value is `false`. If `isDefaultGateway` is set to `true`, then `isGateway` is also set to `true` automatically.
+|Optional: Set to `true` to configure the bridge as the default gateway for the virtual network. The default value is `false`. If `isDefaultGateway` is set to `true`, then `isGateway` is also set to `true` automatically.
 
 |`forceAddress`
 |`boolean`
-|Set to `true` to allow assignment of a previously assigned IP address to the virtual bridge. When set to `false`, if an IPv4 address or an IPv6 address from overlapping subsets is assigned to the virtual bridge, an error occurs. The default value is `false`.
+|Optional: Set to `true` to allow assignment of a previously assigned IP address to the virtual bridge. When set to `false`, if an IPv4 address or an IPv6 address from overlapping subsets is assigned to the virtual bridge, an error occurs. The default value is `false`.
 
 |`hairpinMode`
 |`boolean`
-|Set to `true` to allow the virtual bridge to send an Ethernet frame back through the virtual port it was received on. This mode is also known as _reflective relay_. The default value is `false`.
+|Optional: Set to `true` to allow the virtual bridge to send an Ethernet frame back through the virtual port it was received on. This mode is also known as _reflective relay_. The default value is `false`.
 
 |`promiscMode`
 |`boolean`
-|Set to `true` to enable promiscuous mode on the bridge. The default value is `false`.
+|Optional: Set to `true` to enable promiscuous mode on the bridge. The default value is `false`.
 
 |`vlan`
 |`string`
-|Specify a virtual LAN (VLAN) tag as an integer value. By default, no VLAN tag is assigned.
+|Optional: Specify a virtual LAN (VLAN) tag as an integer value. By default, no VLAN tag is assigned.
+
+|`vlanTrunk`
+|`list`
+|Optional: Assign a VLAN trunk tag. The default value is `none`.
 
 |`mtu`
 |`string`
-|Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
 
+|`enabledad`
+|`boolean`
+|Optional: Enables duplicate address detection for the container side `veth`. The default value is `false`.
+
+|`macspoofchk`
+|`boolean`
+|Optional: Enables mac spoof check, limiting the traffic originating from the container to the mac address of the interface. The default value is `false`.
```

Comment:
you are missing the param preserveDefaultVlan

Reply:
added